web/app/models/DrivingSession.js

```diff
@@ -0,0 +1,14 @@
+// grab the mongoose module
+var mongoose = require("mongoose");
+
+// define our driving session model
+// module.exports allows us to pass this to other files when it is called
+var DrivingSessionSchema = new mongoose.Schema({
+    startTime: { type: Number, required: true },
+    endTime: { type: Number, required: true },
+    distance: { type: Number, required: true },
+    duration: { type: Number, required: true },
```

Comment:
Agreed, since we won't be storing paths no need for a schema for it. I think that would be part of the endpoint issue.